### PR TITLE
Re-order the articles

### PR DESCRIPTION
**Pull request by juntao**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [PR summary agent](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/user-guide/apps/translation-agent.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 9
+sidebar_position: 103
```
Comment (alabulei1) [inline docs/user-guide/apps/translation-agent.md:2]: Do you mean 13?

File: docs/user-guide/apps/llamaedgebook.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 7
+sidebar_position: 105
```
Comment (alabulei1) [inline docs/user-guide/apps/llamaedgebook.md:2]: Do you mean 15?

File: docs/user-guide/apps/gpt-planner.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 10
+sidebar_position: 104
```
Comment (alabulei1) [inline docs/user-guide/apps/gpt-planner.md:2]: Do you mean 14?

File: docs/user-guide/apps/continue.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 11
+sidebar_position: 102
```
Comment (alabulei1) [inline docs/user-guide/apps/continue.md:2]: Do you mean 12?

File: docs/user-guide/apps/llamaedgebook.md (hunk 2 above)
Comment (juntao) [reply docs/user-guide/apps/llamaedgebook.md:2]: No. I want to leave 1-100 for articles about frameworks. And 100+ for articles about applications. 

This way, I do not need to renumber all the articles if I need to add one at the top.

**Pull request merged**